### ISSUE TITLE
Yapf-diff not installed correctly

### ISSUE THREAD

**Issue by myw** [google/yapf#920]
To reproduce, on a clean system:

```
[me@localhost] $ pip install -U yapf
[me@localhost] $ yapf-diff -h
Traceback (most recent call last):
  File "/Users/me/.pyenv/versions/3.7.10/bin/yapf-diff", line 5, in <module>
    from yapf.third_party.yapf_diff.yapf_diff import main
ModuleNotFoundError: No module named 'yapf.third_party'
```

This is happening because of an error in the `setup.py` file: `yapf.third_party.yapf_diff` was not added to the `packages` kwarg to `setup()`, so the files are not actually getting installed.

I'm pushing a PR that fixes it in a minute.

Surprising no one has brought it up yet, but ¯\_(ツ)_/¯ . I'm just happy to see it fixed.

**Comment by gwelymernans** [comment]
Yikes! Thanks for supplying a patch for it. :-)

**Comment by d3sm0-ctrl** [comment]
This issue does not seem solved in the released version on `pip` and `conda`. On a brand new env, after `pip install yapf` i get:
```
Traceback (most recent call last):
  File "/home/user/miniconda3/envs/venv/bin/yapf-diff", line 5, in <module>
    from yapf.third_party.yapf_diff.yapf_diff import main
ModuleNotFoundError: No module named 'yapf.third_party'
```
While installing from git solves this issue.

Could you please check?

**Comment by caufman** [comment]
I'm facing the same problem:

```
(venv) $> yapf-diff ./src/general_models.py 
Traceback (most recent call last):
  File "/home/user/path/to/venv/bin/yapf-diff", line 5, in <module>
    from yapf.third_party.yapf_diff.yapf_diff import main
ModuleNotFoundError: No module named 'yapf.third_party'
```

**Comment by azec-pdx** [comment]
Would like to report this is fixed with `yapf-0.32.0` for me.

Before updating:
```bash
$ yapf-diff -h
Traceback (most recent call last):
  File "/Users/amer/.pyenv/versions/infra-lambda-ses-notifications-handler/bin/yapf-diff", line 5, in <module>
    from yapf.third_party.yapf_diff.yapf_diff import main
ModuleNotFoundError: No module named 'yapf.third_party'
```

...after updating ...
```bash
$ pip install -U yapf
Requirement already satisfied: yapf in /Users/amer/.pyenv/versions/3.9.12/envs/infra-lambda-ses-notifications-handler/lib/python3.9/site-packages (0.31.0)
Collecting yapf
  Downloading yapf-0.32.0-py2.py3-none-any.whl (190 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 190.2/190.2 KB 4.9 MB/s eta 0:00:00
Installing collected packages: yapf
  Attempting uninstall: yapf
    Found existing installation: yapf 0.31.0
    Uninstalling yapf-0.31.0:
      Successfully uninstalled yapf-0.31.0
Successfully installed yapf-0.32.0

$ yapf-diff -h
usage: yapf-diff [-h] [-i] [-p NUM] [--regex PATTERN] [--iregex PATTERN] [-v] [--style STYLE] [--binary BINARY]

This script reads input from a unified diff and reformats all the changed
lines. This is useful to reformat all the lines touched by a specific patch.
Example usage for git/svn users:

  git diff -U0 --no-color --relative HEAD^ | yapf-diff -i
  svn diff --diff-cmd=diff -x-U0 | yapf-diff -p0 -i

It should be noted that the filename contained in the diff is used unmodified
to determine the source file to update. Users calling this script directly
should be careful to ensure that the path in the diff is correct relative to the
current working directory.

optional arguments:
  -h, --help            show this help message and exit
  -i, --in-place        apply edits to files instead of displaying a diff
  -p NUM, --prefix NUM  strip the smallest prefix containing P slashes
  --regex PATTERN       custom pattern selecting file paths to reformat (case sensitive, overrides -iregex)
  --iregex PATTERN      custom pattern selecting file paths to reformat (case insensitive, overridden by -regex)
  -v, --verbose         be more verbose, ineffective without -i
  --style STYLE         specify formatting style: either a style name (for example "pep8" or "google"), or the name of a file with style settings. The default is pep8
                        unless a .style.yapf or setup.cfg file located in one of the parent directories of the source file (or current directory for stdin)
  --binary BINARY       location of binary to use for yapf
```
